app/Exports/UsersExport.php

```diff
@@ -7,20 +7,32 @@
 use App\Exports\UsersSheets\StatusesExport;
 use App\Exports\UsersSheets\StudentsCouncilFeedback;
 use App\Models\Role;
+use App\Models\User;
+use Illuminate\Database\Eloquent\Builder;
 use Maatwebsite\Excel\Concerns\WithDefaultStyles;
 use Maatwebsite\Excel\Concerns\WithMultipleSheets;
 use PhpOffice\PhpSpreadsheet\Style\Style;
 
 class UsersExport implements WithMultipleSheets, WithDefaultStyles
 {
+    private Builder|User $includedUsersQuery;
+
+    public function __construct(Builder|User $includedUsersQuery)
```

Comment:
why don't you pass the User collection? Using builders here makes it more complex. Order the query by name beforehand and you will have the same result.

Reply:
Done!